.github/PR_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-XXX
+
+## Checklist
+
+I have:
+
+- [ ] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
+- [ ] **Tested** my feature.
```

Comment:
"Tested my feature" est ambigu

je propose d'énumérer les 2 actions:
- Tested my feature on simulator
- Tested my feature on relevant devices (depending on the way we define what is "DONE" for devices)

Reply:
Ok, j'entendais plutôt le fait d'écrire des tests. 
J'ai retiré "testted my feature", et à la place j'ai mis deux nouveaux points :
- Écrire des test unitaires
- Valider la feature sur les device (réel ou virtuel) appropriés
Bien sûr on pourra MaJ le template au fur et à mesure. 
L'objectif est ici de valider les grandes lignes ;).